lib/stack_master/parameter_resolver.rb

```diff
@@ -27,19 +27,53 @@ def resolve
 
     private
 
+    def require_parameter_resolver(file_name)
+      begin
+        require "stack_master/parameter_resolvers/#{file_name}"
+      rescue LoadError
+        raise ResolverNotFound.new(file_name)
+      end
+    end
+
+    def load_parameter_resolver(class_name)
+      begin
+        # Check if the class name already exists
+        return if resolver_class_const(class_name)
```

Comment:
Any reason for the conditional return? I think it would work without it, the difference being the return value set to the constant instead of nil.